content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
```

Comment:
That's not a bad way to handle it, but in that case we need to message it carefully on the user-facing docs too.